### DISCUSSION TITLE
how can i contribute?

### DISCUSSION THREAD

**Discussion by rullyalves** [ethanblake4/flutter_eval#48] [General]
I would like to contribute to the project, what would be the steps?

**Comment by ethanblake4** [comment]
Thanks for your interest! The easiest way to contribute right now would probably be writing tests or examples. If you want to get a little deeper in the project you can write new bindings for Flutter widgets, it should be possible to figure out how to do this by looking at the code of flutter_eval and its commits. Let me know if you have any technical issues or questions.
